node_scenarios.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import time
+import json
+import boto3
```

Comment:
boto3 module needs to be listed as a dependency by adding it to requirements.txt file.

Reply:
We get boto3 automatically with AWS CLI installation.

Reply:
Maybe add awscli to the requirements.txt then?